### PR TITLE
Documentation for logging with Elasticsearch and Kibana

### PR DESCRIPTION
**Pull request by satnam6502**
This adds a new document that focuses on cluster level logging with Elasticsearch and Kibana.

### CODE REVIEW

File: docs/getting-started-guides/logging-elasticsearch.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+```
+
+The Elasticsearch and Kibana services are not directly exposed via a publically reachable IP address. Instead,
+they can be accessed via the service proxy running at the master. The URLs for accessing Elasticsearch
+and Kibana via the service proxy can be found using the `kubectl cluster-info` command.
+
+```
+$ kubectl cluster-info
+Kubernetes master is running at https://146.148.94.154
+Elasticsearch is running at
```
Comment (a-robinson) [inline docs/getting-started-guides/logging-elasticsearch.md]: Why no link here?

File: docs/getting-started-guides/logging-elasticsearch.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+Elasticsearch is running at 
+Kibana is running at https://146.148.94.154/api/v1/proxy/namespaces/default/services/kibana-logging
+KubeDNS is running at https://146.148.94.154/api/v1/proxy/namespaces/default/services/kube-dns
+Grafana is running at https://146.148.94.154/api/v1/proxy/namespaces/default/services/monitoring-grafana
+Heapster is running at https://146.148.94.154/api/v1/proxy/namespaces/default/services/monitoring-heapster
+InfluxDB is running at https://146.148.94.154/api/v1/proxy/namespaces/default/services/monitoring-influxdb
+
+```
+
+Before accessign the logs ingested into Elasticsearch using a browers and the service proxy URL we need to find out
```
Comment (a-robinson) [inline docs/getting-started-guides/logging-elasticsearch.md]: a couple spelling errors

File: docs/getting-started-guides/logging-elasticsearch.md (hunk 1 above)
Comment (satnam6502) [inline docs/getting-started-guides/logging-elasticsearch.md]: Fixed (after checking that this was an editor fumble and that cluster-info does indeed do the right thing).

File: docs/getting-started-guides/logging-elasticsearch.md (hunk 2 above)
Comment (satnam6502) [inline docs/getting-started-guides/logging-elasticsearch.md]: Fixed the spelling errors here and elsewhere.

**Comment by satnam6502** [comment]
PTAL. Thank you.

**Comment by a-robinson** [comment]
LGTM

**Pull request merged**